API.md

```diff
@@ -1947,7 +1947,9 @@ If there was an error, expect an HTTP status code in either the 4XX or 5XX range
 
 Creates a new group.
 
-This operation requires rodsadmin level privileges.
+This operation requires rodsadmin or groupadmin level privileges.
+
+If the user is of type groupadmin, they must be a member of the target group to execute this operation.
```

Comment:
you can remove this line.  can't be a member of a not-yet-created-group.

Reply:
Will fix that right up.

Reply:
done